### ISSUE TITLE
Import from wigle.net to import all stats

### ISSUE THREAD

**Issue by mtbrandao** [wiglenet/wigle-wifi-wardriving#10]
Today, the import observed only imports the BSSID.

It would be logical to import all data to the database (SSID, WEP, lat, long, channel, etc).
Since I can see it on wigle site via the query funcion
http://wigle.net/gps/gps/main/query/

Someone could do a script to import "Only Networks I Was the First to Discover" into the sqllite file.
Of couse, this takes a lot of work and CPU stress.

**Comment by rksh** [comment]
note: uploads page buttons now permit specific upload import from WiGLE, which partially resolves this issue.

**Comment by slammingprogramming** [comment]
Yeah, I'll bite on this one too. I'll look into implementing pagination logic and rate-limiting with exponential back-off for rate limits and other errors to ensure this doesn't bash the servers too hard.

**Comment by rksh** [comment]
please be aware that this is something of an anti-pattern: importing all previous observations from other devices will slow database performance (and by extension scanning and other aspects of application peformance) proportionally on your phone. We've said this elsewhere, but you shouldn't try to centralize _all_ your observations on a single phone - rather you should use the "export SQLite" options and merge on a laptop/desktop machine where you're not limited by mobile processing and battery life.

Details of how to merge database files from the SQLite prompt are published on [our Tools and Downloads page](https://wigle.net/tools).

**Comment by slammingprogramming** [comment]
I see, that does sound like a better solution. Do you think it would be reasonable to implement this in a server-client manner for use cases? IIRC I think I saw something like this where you can query the WiGLE.net db for specific MACs, unless I'm mistaken, making this essentially pointless if one can do this on the go already in the most useful manner?

**Comment by rksh** [comment]
I don't think I understand the question.

- You can always export your database from the app (if you're keen to keep your specific observations, this is a good way to do it) -this makes a good backup mechanism.
- You can query the site for the networks you've uploaded, but that doesn't re-vend your specific observations
- You can re-request a particular transid you've uploaded from the site, but this will result in a sanitized, date-time-truncated list of observations, we don't keep a copy of the exact data you upload

It's just generally not a good idea to load ALL your lifetime observations onto a single phone, since that will slow the database performance on that phone.
